### ISSUE TITLE
[Date input, Memorable date, Number input, Select, Text input, Textarea] - Increase width of error state (red) borders from 2px to 4px

### ISSUE THREAD

**Issue by GnatalieH** [department-of-veterans-affairs/vets-design-system-documentation#960]
## Description
In order to maintain alignment with USWDS and a11y best practices, we should increase the border width from 2px to 4px on all form controls that use the red border on error states. This should be completed after the discovery work in ticket #1015 has been done.

Example using 4px border:
![Screen Shot 2022-06-16 at 18.30.18.png](https://images.zenhubusercontent.com/6111980ab6b167c27c9a5723/9065da88-7708-4c62-a807-64e078d03063)

## Details
The following components should be updated:
- [x] Date input
- [x] Memorable date
- [x] Number input
- [x] Select
- [x] Text input
- [x] Textarea

## Tasks
- [x] Review DST backlog for outstanding design issues with this component, if necessary
- [x] Create designs for component
- [x] Review designs with PO and/or DSC
- [x] Review designs with an accessibility specialist
- [x] Review designs with DST members (Carol can help schedule this)
- [x] Address any comments from reviews, if necessary
- [x] Comment on this ticket with any accessibility considerations engineers may need to know
- [x] Comment on this ticket with a link to the designs and post in DST Slack channel

## Acceptance Criteria
- [x] Component design is complete and has been reviewed
- [x] Accessibility considerations have been added to this ticket, if necessary
- [x] Link to design has been added to this ticket and shared in Slack

**Comment by caw310** [comment]
As discussed in planning for Sprint 5, @GnatalieH , please break up this work and write another ticket. Thanks.

**Comment by andresriveratoro** [comment]
The development for updating the border width from 2px to 4px has been completed. If there are any changes to padding, we will need a development issue for that work.

**Comment by GnatalieH** [comment]
[Here is a link](https://www.sketch.com/s/1cb56b0b-df7c-456b-a8e4-f08a3553a5d3) to the work I've done on our form control error states, including increasing the input field border from 2px to 4px.

**Comment by humancompanion-usds** [comment]
@GnatalieH - Looks great! You have some Symbols for states that are not on your main artboard. For example, Form / text input / hint text-open-error. Will you be adding those as well? I think they are useful examples and worth adding.

**Comment by GnatalieH** [comment]
@humancompanion-usds thanks! Yes, I'll add those symbols now. I'm going to move everything to the main Pattern library file now as well.

**Comment by GnatalieH** [comment]
Closing as completed as I've added the examples Matt suggested.
